foreman-installer/foreman-installer.spec

```diff
@@ -58,11 +58,14 @@ rake install \
   SYSCONFDIR=%{buildroot}%{_sysconfdir} \
   --trace
 
+%post
+foreman-installer --scenario foreman --migrations-only > /dev/null
```

Comment:
Is specifying the scenario here going to cause any problems if more are added to the package?